test/charm_test.py

```diff
@@ -31,33 +23,6 @@
 import domain
 
 
-class CharmTest(unittest.TestCase):
```

Comment:
What was the reason of removing those tests?

Reply:
Pure laziness. I'll replace it with something more appropriate over the weekend. ;-) Or next week.

Reply:
I vote for next week ;-) It is very important to get some rest!

Reply:
Alrighty so I'm permanently deleting this hackish way of check if the Charm initializes properly in favor of that test-harness-based test that you've commented out temporarily due to some Harness issues as you've mentioned in the comments on this file.